External/AvalonTools/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 if(RDK_BUILD_AVALON_SUPPORT)
+set(AVALONTOOLS_DIR "$ENV{AVALONTOOLS_DIR}") 
```

Comment:
This block of changes screws up subsequent runs of cmake in an already existing build directory where RDK_BUILD_AVALON_SUPPORT is already set if there is no AVALONTOOLS_DIR variable in the active environment.

Why do you think these are necessary?